AIPscan/Data/data.py

```diff
@@ -224,6 +254,84 @@ def derivative_overview(storage_service_id):
     return report
 
 
+def _format_versions_count_query(storage_service_id, start_date=None, end_date=None):
+    """Fetch information from database on format versions.
+
+    :param storage_service_id: Storage Service ID.
+    :start_date: Optional inclusive AIP creation start date
+    (datetime.datetime object).
+    :end_date: Optional inclusive AIP creation end date
+    (datetime.datetime object).
```

Comment:
```suggestion
    :param storage_service_id: Storage Service ID.
    :start_date: Optional inclusive AIP creation start date
        (datetime.datetime object).
    :end_date: Optional inclusive AIP creation end date
        (datetime.datetime object).
```
I am still getting used to this more complex style of docstring. Admittedly I haven't spent a lot of time looking at this, but what do you think about adding a hanging indent: https://stackoverflow.com/a/50455731 - to help readability? It would be good to do that wherever we break a docstring over two lines. Initially it's hard to pick up as part of the same string. 

Is there another idiomatic approach that you happen to know about?

Reply:
These detailed docstrings are a new-ish habit for me so I don't have a ready answer for other approaches but this is a good suggestion and I agree it would help a lot with readability.